queries.sql

```diff
@@ -158,3 +158,70 @@ GROUP BY full_name
 ORDER BY c DESC
 FETCH FIRST 1 ROWS WITH TIES;
 
+
+
+-- How many different animals did Stephanie Mendez see?
+
+SELECT Count(*) FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id
+WHERE vets.name = 'Stephanie Mendez';
+
+-- List all vets and their specialties, including vets with no specialties.
+
+SELECT vets.name , species.name FROM vets
+JOIN specializations ON vets.id = specializations.vets_id
+JOIN species ON species_id = species.id;
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+
+SELECT animals.name FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id
+WHERE vets.name = 'Stephanie Mendez' AND visits.visite_date BETWEEN '04-01-2020' AND '08-30-2020';
+
+-- What animal has the most visits to vets?
+
+SELECT animals.name, Count(*) As c FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id
+GROUP BY animals.name
+ORDER BY c DESC
+FETCH FIRST 1 ROWS WITH TIES;
+
+-- Who was Maisy Smith's first visit?
+
+SELECT animals.name , visits.visite_date , vets.name FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id
+WHERE vets.name = 'Maisy Smith'
+ORDER BY visite_date
+FETCH FIRST 1 ROWS WITH TIES;
+
+-- Details for most recent visit: animal information, vet information, and date of visit.
+
+SELECT * FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id
+ORDER BY visite_date DESC
+FETCH FIRST 1 ROWS WITH TIES;
```

Comment:
- [ ] To get the `Details for the most recent visit` the SELECT statement should return animal information, vet information, and date of visit. e.g.
```sql
SELECT animals.*, vets.*, visits.visit_date
FROM visits
LEFT JOIN animals ON animals.id = visits.animal_id
LEFT JOIN vets ON vets.id = visits.vet_id
ORDER BY visits.visit_date DESC
LIMIT 1;
```

Reply:
Thank you @ZeenatLawal .Am implementing the required changes.